lib/publiccloud/instance.pm

```diff
@@ -209,16 +209,20 @@ sub wait_for_ssh
     $args{timeout}            //= 600;
     $args{proceed_on_failure} //= 0;
     $args{username}           //= $self->username();
+    $args{migration}          //= 0;
     my $start_time = time();
 
     # Check port 22
-    while ((my $duration = time() - $start_time) < $args{timeout}) {
+    while ((my $duration = time() - $start_time) < ($args{timeout} + $args{migration})) {
```

Comment:
not sure why we need special time length here . This code currently successfully initiate SLE 12 SP5 w/o any extra time above https://openqa.suse.de/tests/5498986 . can you share what kind of failure you got before you added this timeout extension ?

Reply:
This was suggested by me, because we were getting a spam of record_info frames in the job that made it understandable

Reply:
example: http://copland.arch.suse.de/tests/214

Reply:
well this problem not related to migration anyhow please see for example https://openqa.suse.de/tests/5474817  ( not so bad but also quite some spam ) . So solving it in this way would be really confusing .  Let's simply drop this from this commit and solve it independently

Reply:
good point, ok to do this in different commit.